### PR TITLE
DOCS-12083-mqtt-publish-EDITORIAL-kt 

### PR DESCRIPTION
**Pull request by kevintroller**
MQTT Connector Publish EDITORIAL

### CODE REVIEW

File: mqtt3/1.0/modules/ROOT/edits/mqtt3-connector-publish.adoc [hunk 1]
```diff
@@ -0,0 +1,61 @@
+= Publish Messages to an MQTT Broker with MQTT Connector
+
+In the following example, you configure the Anypoint Connector for MQTT (MQTT Connector) *Publish* operation that enables you to publish a message to the desired topic and with a specific quality of service (QoS).
```
Comment (kbostrom) [inline mqtt3/1.0/modules/ROOT/edits/mqtt3-connector-publish.adoc]:
```suggestion
In the following example, you configure the Anypoint Connector for MQTT (MQTT Connector) *Publish* operation, which enables you to publish a message to the desired topic and with a specific quality of service (QoS).
```

File: mqtt3/1.0/modules/ROOT/edits/mqtt3-connector-publish.adoc [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
```
Comment (kbostrom) [inline mqtt3/1.0/modules/ROOT/edits/mqtt3-connector-publish.adoc]: Ditto previous edits in related PRs regarding shorter wording options.

File: mqtt3/1.0/modules/ROOT/edits/mqtt3-connector-publish.adoc [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+* *Client ID*: `smart-bentley-123`
+* *Username*: `usertest`
+* *Password*: `passtest`
+* *URL*: `tcp://127.0.0.1:1883`
+[start=13]
+. Click *OK*.
+. In the *Publish* configuration screen, set the following fields:
++
+* *Topic*: `quotes/terryPratchett`
+* *Message* `#["Stories of imagination tend to upset those without one."]`
```
Comment (kbostrom) [inline mqtt3/1.0/modules/ROOT/edits/mqtt3-connector-publish.adoc]: lol

Comment (kbostrom) [reply mqtt3/1.0/modules/ROOT/edits/mqtt3-connector-publish.adoc]: OK, I would be remiss if I did not caution you to carefully consider your examples for global impact (examples that you know people around the world will likely understand, rather than regional or cultural examples).
However, Terry Pratchett....

File: mqtt3/1.0/modules/ROOT/edits/mqtt3-connector-publish.adoc [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    <http:listener config-ref="HTTP_Listener" path="/mqtt3/publish" />
+    <mqtt3:publish config-ref="MQTT_Config" topic="quotes/terryPratchett" qos="AT_LEAST_ONCE" isRetained="true">
+        <mqtt3:message>#["Stories of imagination tend to upset those without one."]</mqtt3:message>
+    </mqtt3:publish>
+</flow>
+----
+
+== Mark Messages as Retained
+
+By default, a broker discards a received message on a topic for which there are no current subscribers. Although, the broker retains the message if you configure the *Is retained* field to `True`. The broker stores the last retained message and the corresponding quality of service for the selected topic. +
```
Comment (kbostrom) [inline mqtt3/1.0/modules/ROOT/edits/mqtt3-connector-publish.adoc]:
```suggestion
By default, a broker discards a received message on a topic for which there are no current subscribers. If you want the broker to retain the message, configure *Is retained* to `True`. The broker stores the last retained message and the corresponding quality of service for the selected topic. +
```

File: mqtt3/1.0/modules/ROOT/edits/mqtt3-connector-publish.adoc [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+Each client that subscribes to a topic pattern that matches the topic of the retained message receives the retained messages after they subscribe. The broker stores only one retained message per topic. This enables new subscribers to receive the most current value rather than waiting for the next update from a publisher.
```
Comment (kbostrom) [inline mqtt3/1.0/modules/ROOT/edits/mqtt3-connector-publish.adoc]: clients subscribe to topic patterns? Or maybe...:
```suggestion
Each client that subscribes to a topic that has the same pattern as that of a retained message receives the retained message after they subscribe. The broker stores only one retained message per topic. This enables new subscribers to receive the most current existing value rather than waiting for the next update from a publisher.
```
Will the audience know what you mean by "pattern"?

**Review by kbostrom — DISMISSED** [review]
CE

**Review by dhirsch1 — APPROVED** [review]
